examples/3d-tiles/tile-3d-layer.js

```diff
@@ -328,10 +308,12 @@ export default class Tile3DLayer extends CompositeLayer {
       return {};
     }
 
-    const {rtcCenter} = tileHeader.content;
-    const {transform} = tileHeader.userData;
+    const {
+      computedTransform,
+      content: {rtcCenter}
+    } = tileHeader;
 
-    let modelMatrix = new Matrix4(transform);
+    let modelMatrix = new Matrix4(computedTransform);
     if (rtcCenter) {
       modelMatrix.translate(rtcCenter);
```

Comment:
Different PR: Time to move these calculations into the 3d-tiles module. The layer should not need to know about rtcCenter...